### PR TITLE
changed color from blue to light red orche.

### PR DESCRIPTION
**Pull request by K4stor**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+}
```
Comment (sphinxc0re) [inline .vscode/settings.json]: Is this file checked in on purpose? Also you are missing a newline.

Comment (K4stor) [reply .vscode/settings.json]: oops... sorry.... will change that

Comment (sphinxc0re) [reply .vscode/settings.json]: You might consider adding this file to the `.gitignore`

**Comment by sphinxc0re** [comment]
Good job 👍

**Pull request merged**